.github/workflows/build.yml

```diff
@@ -134,6 +134,7 @@ jobs:
       matrix:
         java_version: [17, 23]
         test_mode: ["test_integration", "test_docs", "test_aws", "test_azure", "test_google", "test_wave"]
+        strict_syntax: [true, false]
```

Comment:
This results in too many builds. I was suggesting to add it as a `test_mode` then hook in the test.sh script and run `test_integration` using the new parser

Reply:
That makes more sense. I didn't want to double the number of builds either